### PR TITLE
Add mysql-client, helm, tfenv+terraform

### PR DESCRIPTION
**Pull request by amulobaev**

### CODE REVIEW

**Comment by vdefeo-xo** [comment]
Your Pull Request has been selected for a project-agnostic code review by [Aurea Central Reviewers Team](https://github.com/orgs/trilogy-group/teams/aurea-central-reviewers/members).

Please consider using [Analyzer UI](https://confluence.devfactory.com/display/CodeReview/Analyzer+UI) extension to catch issues in pull requests.

<details><summary>More information</summary>

_This is an initiative to improve overall code quality. If your change is **urgent**, you **don't** have to wait for our review._ 

_NOTE: This review **does not** replace your team's review._

</details>

File: full/Dockerfile [hunk 1]
```diff
@@ -210,17 +210,32 @@ RUN bash -lc "cargo install cargo-watch cargo-edit cargo-tree"
...
 RUN sudo apt-get update && \
     sudo apt-get install -y apt-transport-https ca-certificates curl && \
     sudo curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg && \
     echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list && \
     sudo apt-get update && \
     sudo apt-get install -y kubectl
+# AWS CDK
 RUN npm install -g aws-cdk
+# mysql-client
+RUN sudo apt-get update && sudo apt-get install -y mysql-client
```
Comment (vdefeo-xo) [inline full/Dockerfile:229]: <details>
<summary>:no_entry_sign: <strong>This RUN command is calling sudo.</strong> (click for details)</summary>

An important improvement would be to remove the `sudo` from the command execution. If it is not possible to run the command with a non-root user, consider using [gosu](https://github.com/tianon/gosu).  
  
This change would ensure that no unpredictable TTY and signal-forwarding behavior will happen.

**[More details](https://confluence.devfactory.com/x/39RnH)**

## The solution in this case
```
RUN apt-get update && apt-get install -y mysql-client
```
</details>

[//]: # (<a id="eaa21eff-8a43-45f5-8ae6-2e27da43d52b" data-finding-id="476566751" href="none" codereview-uuid-reserved-tag data-business-id="5a8cd580-aa5f-482a-bfc7-a840f35c255b" data-analyzer="cap" />)

File: full/Dockerfile [hunk 2]
```diff
@@ -210,17 +210,32 @@ RUN bash -lc "cargo install cargo-watch cargo-edit cargo-tree"
...
+# helm
+RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 && chmod 700 get_helm.sh && ./get_helm.sh && rm get_helm.sh
+# tfenv + terraform
+RUN git clone https://github.com/trilogy-group/tfenv.git ~/.tfenv && \
```
Comment (vdefeo-xo) [inline full/Dockerfile:233]: <details>
<summary>:no_entry_sign: :repeat: <strong>This RUN command is calling sudo.</strong> (click for details)</summary>
First occurrence of this finding in this pull request can be found <a href="https://github.com/trilogy-group/workspace-images/pull/3#discussion_r661625507">here</a></details>

[//]: # (<a id="79e7cc60-d9be-11eb-8ef2-6389bd893c9b" data-business-id="5a8cd580-aa5f-482a-bfc7-a840f35c255b" href="none" data-finding-id="476566751" repeated-comment codereview-uuid-reserved-tag />)

**Review by vdefeo-xo — COMMENTED** [review]
Hi, I am the IaC expert who performed the review of this Pull Request. We have well-defined [coding and review standards](https://confluence.devfactory.com/display/CodeReview/Findings), and you will find below suggestions to improve the code quality, based on these guidelines.
My additional comments are in the Notes section. I hope you find them useful.

<table>
  <tr>
    <td>:no_entry_sign:<b>Severes:</b>  2</td>
    <td>:exclamation:<b>Reds:</b>  0</td>
    <td>:warning:<b>Yellows:</b>  0</td>
    <td>:white_check_mark:<b>Greens:</b>  0</td>
  </tr>
</table>

<details><summary>More information</summary>
  <table>
    <tr>
      <td>:no_entry_sign:</td>
      <td><b>Severe Findings</b> point out serious violations which can affect stability, performance, maintainability and therefore should be fixed.
        <ul>
          <li>Please note that the merge might have been blocked until all Severe Findings are fixed.</li>
          <li>While it is highly recommended to implement all findings only Severe Findings might have been blocking the merge</li>
        </ul>
      </td>
    </tr>
    <tr>
      <td>:exclamation:</td>
      <td><b>Red Findings</b> represent potential bugs and it is highly recommended to implement them.<br/>
      Not required for the approval of the PR
      </td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td><b>Yellows</b> are suggestions to improve the code, warnings and tips<br/>
      Not required for the approval of the PR, and should not be used for FTAR calculation.
      </td>
    </tr>
    <tr>
      <td>:white_check_mark:</td>
      <td><b>Green Findings</b> highlight good ideas and solutions</td>
    </tr>
    <tr>
      <td>:repeat:</td>
      <td><b>Repeated</b> instances of the same findings are counted just once and flagged with the repeat icon</td>
    </tr>
  </table>

  Feel free to reply to this comment, (make sure you `@tag` us) so we can start a conversation. The goal here is to learn and grow together and make our products better.

  > NOTE: This review **does not** replace your team's review._
</details>
<details>
    <summary>T4W FTAR Statistics</summary>
    <br />
    CodeRead team introduced new quality measurement to evaluate the trailing 4 weeks First Time Acceptance Rate of Pull Requests, this PR decreased your FTAR from 85.71 % to 75.00 %, so we recommend studying raised finding to improve the quality of written code for next upcoming PRs.
	<br />
    -
    <details>
        <summary>More Details</summary>
        <br />
        <table>
            <tr>
                <td>Current FTAR</td>
                <td><b>75.00 %</b></td>
            </tr>
            <tr>
                <td>Total Reviewed Pull Requests for the last 4 weeks</td>
                <td>8</td>
            </tr>
            <tr>
                <td>Passed</td>
                <td>6</td>
            </tr>
            <tr>
                <td>Rejected</td>
                <td>2</td>
            </tr>
        </table>
        More details about how FTAR is calculated, refer to this page - <a
            href="https://confluence.devfactory.com/x/Cw8dGg">FTAR - First time acceptance rate</a>
    </details>

</details>

**Review by pavan-tri — APPROVED** [review]

**Pull request merged**